src/Lecture4.hs

```diff
@@ -222,8 +269,12 @@ Have a look at the 'sconcat' function from the 'Semigroup' typeclass to
 implement the next task.
 -}
 
+
+strictSconcat:: (Semigroup a) => NonEmpty a -> a
+strictSconcat = foldr1 (\x !acc -> x <> acc)
```

Comment:
That's a nice implementation using `foldr1` function 👏🏻 
Alternatively, you can pattern match on `NonEmpty` and use `foldl'`
```suggestion
strictSconcat (x :| xs) = foldl' (<>) x xs
```